src/ui/views/TeamHistory/index.tsx

```diff
@@ -59,7 +59,19 @@ const TeamHistory = ({
 					Schedule
 				</a>{" "}
 				|{" "}
-				<a href={helpers.leagueUrl(["news", `${abbrev}_${tid}`])}>News Feed</a>
+				<a href={helpers.leagueUrl(["news", `${abbrev}_${tid}`])}>News Feed</a>{" "}
+				|{" "}
+				<a href={helpers.leagueUrl(
+						season === undefined ? ["player_stats"] : [
+									"player_stats",
+									`${abbrev}_${tid}`,
+									"career",
+									process.env.SPORT === "basketball" ? "totals" : statType,
+							  ],
+					)}
+				>
+					{season === undefined ? "Per Game" : "Franchise Leaders"}
```

Comment:
`season` is always defined, it contains the current season. So this should probably just be `"Franchise Leaders"`